src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -416,15 +425,15 @@ static int _PaPulseAudio_ProcessAudio(PaPulseAudio_Stream *stream,
                                        hostFramesPerBuffer );
         }
 
-        /* Is output is enable and buffer data is not NULL
-         * then crap pointer to output ringbuffer
+        /* If output is enabled and the buffer data is not NULL,
+         *  a pointer to the output ring buffer is assigned.
```

Comment:
Does this capture the intended meaning? It reads very strangely.

Reply:
Seems to be updated already.

Reply:
I'm just going to delete it.